src/bower_components/emby-webcomponents/globalize.js

```diff
@@ -62,6 +63,11 @@ define(['connectionManager', 'userSettings', 'events'], function (connectionMana
         for (var i in allTranslations) {
             ensureTranslation(allTranslations[i], culture);
         }
+        if (culture !== fallbackCulture) {
```

Comment:
Why not have the if inside the loop instead?

Reply:
Because speed! :) This way it only checks once instead of how many entries there is in `allTranslations`.

Reply:
I doubt looping over all the entries twice is faster. Maybe it's faster if your locale is already `en-us`, but it seems like it would be worse for any other case. :wink:

Reply:
Isn't it a moot point? :) This code executes once the application is loaded, and once a localization language is changed. Even if it adds like 50 extra milliseconds is it that bad? :)

For real, it's impossible to tell which one is faster without benchmarks, and I don't think we should really be concerned with performance here.

Reply:
I won't hold this up, but the double `for` loop still makes me cringe. :smile: